src/index.ts

```diff
@@ -224,6 +249,37 @@ export class AthenaNodeClient {
         return query.results;
     }
 
+
+    /**
+     * Get statistics from a query execution
+     *
+     * @private
+     * @template T
+     *
+     *
+     * @returns {Promise<T[]>} - parsed query result rows
+     * @memberof AthenaNodeClient
+     * @param {string} executionId
+     */
+    private async getQueryStatistics(executionId: string): Promise<Statistics> {
+        const input: GetQueryExecutionCommand = new GetQueryExecutionCommand({
+            QueryExecutionId: executionId
+        });
+
+        const response = await this._client.send(input);
+        const bytes = response?.QueryExecution?.Statistics?.DataScannedInBytes ?? null;
+        const timeInSeconds = response?.QueryExecution?.Statistics?.EngineExecutionTimeInMillis ?? 0 / 1000
+
+        const stats = new Statistics();
+
+        if (bytes != null) {
+            stats.dataScannedInBytes = bytes;
+            stats.executionTimeInSeconds = timeInSeconds;
+        }
+
+        return stats;
```

Comment:
Al pasar a ser una interfaz, y poner bytes ya con valor por defecto (que yo lo pondría a undefined en vez de null), puedes construir el objeto cumpliendo con la interfaz (ya defines el tipo en el return de la función) y queda más límpio.

```suggestion
        return {
            dataScannedInBytes: bytes,
            executionTimeInSeconds: timeInSeconds,
        };
```